pkg/solana/config/toml.go

```diff
@@ -173,19 +163,36 @@ func setFromChain(c, f *solcfg.Chain) {
 	if f.MaxRetries != nil {
 		c.MaxRetries = f.MaxRetries
 	}
+	if f.FeeEstimatorMode != nil {
+		c.FeeEstimatorMode = f.FeeEstimatorMode
+	}
+	if f.ComputeUnitPriceMax != nil {
+		c.ComputeUnitPriceMax = f.ComputeUnitPriceMax
+	}
+	if f.ComputeUnitPriceMin != nil {
+		c.ComputeUnitPriceMin = f.ComputeUnitPriceMin
+	}
+	if f.ComputeUnitPriceDefault != nil {
+		c.ComputeUnitPriceDefault = f.ComputeUnitPriceDefault
+	}
+	if f.FeeBumpPeriod != nil {
+		c.FeeBumpPeriod = f.FeeBumpPeriod
+	}
 }
 
 func (c *TOMLConfig) ValidateConfig() (err error) {
+	errA := []error{}
+
 	if c.ChainID == nil {
-		err = errors.Join(err, config.ErrMissing{Name: "ChainID", Msg: "required for all chains"})
+		errA = append(errA, config.ErrMissing{Name: "ChainID", Msg: "required for all chains"})
 	} else if *c.ChainID == "" {
-		err = errors.Join(err, config.ErrEmpty{Name: "ChainID", Msg: "required for all chains"})
+		errA = append(errA, config.ErrEmpty{Name: "ChainID", Msg: "required for all chains"})
 	}
 
 	if len(c.Nodes) == 0 {
-		err = errors.Join(err, config.ErrMissing{Name: "Nodes", Msg: "must have at least one node"})
+		errA = append(errA, config.ErrMissing{Name: "Nodes", Msg: "must have at least one node"})
 	}
-	return
+	return errors.Join(errA...)
```

Comment:
Is this deferred join still necessary?

Reply:
It looks like the internal types will superficially be different, but if we are just using `Unwrap() []error`, then we still see the same slice: https://go.dev/play/p/65QFKJrpODU

Reply:
yeah we are really just using `Unwrap() []error` but I think everything is passed around via the `error` interface? 

this seems like the simplest way to meet that interface without implementing a new type

Reply:
I mean that there should be no difference. Are you certain this is necessary?

Reply:
If you mean that we depend on _always_ having something that implements `Unwrap() []error`, then couldn't we can relax that assumption and do a conversion for the base cases of 0/1 on the other side?

Reply:
update: this is the only way that works right now - doing a recursive unwrap within core does not solve the issue

Reply:
revisiting in a future ticket: https://smartcontract-it.atlassian.net/browse/BCI-3330